Classes/Place.py

```diff
@@ -5,3 +5,69 @@ def __init__(self):
 		self.Description = ''
 		self.Name = ''
 		self.CanHaveBuildings = False
+	
```

Comment:
Lot of inconsistent spacing between defs.

Reply:
The spaces have been fixed as well.